packages/headless/src/utils/highlight.ts

```diff
@@ -72,3 +72,46 @@ export function highlightString(params: HighlightParams): string {
   }
   return highlighted;
 }
+
+/**
+ * Add delimiters to a highlighted suggestion.
+ *
+ * @param highlightedSuggestion The highlighted suggestion
+ * @param openingMatchDelimiter The opening delimiter for a match in the suggestion string (e.g. '<strong>')
+ * @param closingMatchDelimiter The closing delimiter for a match in the suggestion string (e.g. '</strong>')
+ * @param openingCorrectedDelimiter The opening delimiter for a correction in the suggestion string (e.g. '<i>')
+ * @param closingCorrectedDelimiter The closing delimiter for a correction in the suggestion string (e.g. '</i>')
+ */
+
+export function formatHighlightedSuggestion(
+  highlightedSuggestion: string,
+  openingMatchDelimiter: string,
+  closingMatchDelimiter: string,
+  openingCorrectedDelimiter: string,
+  closingCorrectedDelimiter: string
+) {
+  if (
+    isEmptyString(openingMatchDelimiter) ||
+    isEmptyString(closingMatchDelimiter) ||
+    isEmptyString(openingCorrectedDelimiter) ||
+    isEmptyString(closingCorrectedDelimiter)
+  ) {
```

Comment:
Not sure about that. This forces the user to give the delimiters, but I was wondering if there could be a use-case were a client only wants to highlight the exact match only and leave the corrected as is and vice-versa (less likely). In that case I was thinking it could check if both pairs are empty and then throw if true. 

Any thoughts?

Reply:
can bueno be used here, leveraging defaults?

Reply:
Following on the comments above, I think it is not nice to force the user to specify all highlights if they are not interested in them. I would not throw an error here, but rather default to an empty string.

It would also be could to structure the properties as a hierarchical object,